app/serializers/api/v1/project_folder_serializer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class ProjectFolderSerializer < ActiveModel::Serializer
+      type :project_folders
+      attributes :id, :name
+
+      belongs_to :team, serializer: TeamSerializer
+      belongs_to :parent_folder, serializer: ProjectFolderSerializer
+    end
```

Comment:
shouldn't it also include projects inside the folder?

Reply:
probably yes...